2-mandatory/6-journey-planner.js

```diff
@@ -123,6 +129,8 @@ function getLocationName() {}
 */
 function journeyPlanner(locations, transportMode) {
   // Implement the function body
+  let placeToGo = locations.filter((location)=> location.includes(transportMode)).map((element)=> element[0])
```

Comment:
how can you use the functions you already wrote in this file to give you the same value?

Reply:
I can call the function doing the same thing.
let placeToGo = locations
    .filter((location) => location.includes(transportMode))
    .map((element) => getLocationName(element));